src/Gui/DlgUnitsCalculatorImp.cpp

```diff
@@ -86,27 +88,34 @@ DlgUnitsCalculator::DlgUnitsCalculator( QWidget* parent, Qt::WindowFlags fl )
           << Base::Unit::ElectricCharge
           << Base::Unit::ElectricCurrent
           << Base::Unit::ElectricPotential
-          << Base::Unit::Frequency
           << Base::Unit::Force
-          << Base::Unit::Stiffness
+          << Base::Unit::Frequency
           << Base::Unit::HeatFlux
+          << Base::Unit::InverseArea
+          << Base::Unit::InverseLength
+          << Base::Unit::InverseVolume
+          << Base::Unit::KinematicViscosity
           << Base::Unit::Length
           << Base::Unit::LuminousIntensity
           << Base::Unit::Mass
           << Base::Unit::MagneticFieldStrength
           << Base::Unit::MagneticFlux
           << Base::Unit::MagneticFluxDensity
           << Base::Unit::Magnetization
-          << Base::Unit::Pressure
           << Base::Unit::Power
+          << Base::Unit::Pressure
+          << Base::Unit::SpecificEnergy
           << Base::Unit::SpecificHeat
-          << Base::Unit::Stress
```

Comment:
Why did you delete this?

Reply:
It was for the reason I mentioned above:

> OK, I have done that for all the units except for:
> CompressiveStrength
> ShearModulus
> Stress
> UltimateTensileStrength
> YieldStrength
> YoungsModulus
> These all show up in the units calculator as 'Pressure' since they share the same units as Pressure, so I have removed them. (They can't be distinguished since Unit objects are presently identified by their fundamental dimensions only, so these objects are all identical to each other.)
> 
>